### DISCUSSION TITLE
Got an unexpected keyword arg 'class_names'

### DISCUSSION THREAD

**Discussion by MananShukla7** [rasbt/mlxtend#1019] [Q&A] [answered]
Hello,
I am trying to plot a confusion matrix with class names but its giving me 
"TypeError: plot_confusion_matrix() got an unexpected keyword argument 'class_names''

**code:**
import mlxtend
from mlxtend.plotting import plot_confusion_matrix
fig,ax=plot_confusion_matrix(conf_m.numpy(),
                             class_names=class_names,
                             figsize=(10,7))
plt.show();

I am working on FashionMNIST dataset

Below is the image of the error:
![image](https://user-images.githubusercontent.com/68244767/222884552-19e0050c-dd88-4fae-8d88-5f2f632fbf74.png)

**Comment by rasbt** [comment, marked as answer]
Hi there,

maybe you are using an older version of MLXtend? Can you share the version you are using? E.g.

```python
import mlxtend
print(mlxtend.__version__)
```

The most current version is 0.21.0 (from Sep 2022)

**Reply by MananShukla7** [reply]
Yeah, I was using the older version..
![image](https://user-images.githubusercontent.com/68244767/222968035-6aba4a80-8c5e-4de3-a4d6-f6c892652b47.png)

Thanks for the suggestion!

**Reply by rasbt** [reply]
Glad this was easily resolved with a version update!
